submissions/israelOladosu/css/contact.css

```diff
@@ -0,0 +1,219 @@
+/* Global stylings */
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+ font-family: "Helvetica Neue", "Open Sans", Helvetica, Arial, sans-serif;
+ color: #5f5f5f;
+}
+img{
+ width:100%;
+}
+/* header */
+header {
+ background: #004b18;
+ display: flex;
+ flex-direction: column;
+ color: white;
+ padding: 1em;
+}
+header .logo{
+ width:10em;
+ margin-left: 0.2em;
+}
+header .logo img {
+ width: 100%;
+ height: auto;
+}
+header ul{
+ list-style: none;
+ margin-left: 0.2em;
+}
+header ul li{
+ padding: 0.6em;
+ margin-left: 2em;
+}
+header ul li a{
+ text-decoration: none;
+ color: white;
+}
+/* main */
+main{
+ display: flex;
+ flex-direction: column;
+ padding: 2em;
+}
+main .col-1{
+ text-align: center;
+}
+main .col-1 .co1-1-row-1 .phone-icon,
+main .col-2 .col-2-row-1 .email
+{
+ width: 2em;
+ margin: 0 auto 0.3em;
+}
+main .col-1 .co1-1-row-1 p:first-of-type,
+main .col-1-row-2 p:first-of-type,
+main .col-1-row-3 p:first-of-type,
+main .col-2 .col-2-row-1 p
+{
+ font-weight: bold;
+}
+main .col-1-row-2 .env{
+ width: 2em;
+ margin: 0.7em auto 0.5em;
+}
+main .col-1-row-3 p:first-of-type{
+ margin: 1em 0 0.2em;
+}
+main .col-2{
+ border: 1px solid #5f5f5f;
+ padding: 1.5em;
+ margin: 1em auto;
+ text-align: center;
+}
+main .col-2 .col-2-row-2 form{
+ width: 100%;
+}
+main .col-2 .col-2-row-2 form input, textarea{
+ margin: 0.5em auto;
+ display: block;
+ padding: 0.7em;
+ width: 100%;
+ border: none;
+ border-bottom: 1px solid #5f5f5f;
+}
+main .col-2 .col-2-row-2 form textarea{
```

Comment:
@Israel-Force Since there is only one text area on the page, you can select it with just `textarea`